trace_decoder/src/compact/compact_prestate_processing.rs

```diff
@@ -122,8 +122,9 @@ pub enum CompactParsingError {
     KeyError(#[from] FromHexPrefixError),
 }
 
+/// CursorBytesErrorInfo
```

Comment:
I noticed this and all other undocumented items got commented like this, i.e.

```rust
/// Foo
struct Foo
```

These are fairly useless, and not having warnings for missing doc will probably mean that these will remain undocumented until someone stumbles upon them and decides to address them.
I'd rather either suggest to allow undocumented items for the time being and remove all these dummy comments, or spend the time to add proper descriptions to help contributors and crate maintainers.

Reply:
> spend the time to add proper descriptions

This gets my vote